ONVIFCamera/Classes/ONVIFCamera.swift

```diff
@@ -228,7 +251,10 @@ public class ONVIFCamera {
     performRequest(request: CameraRequest.getStreamURI(params: params), response: { (result) in
       guard let body = result["Body"] as? [String: Any],
         let response = body["GetStreamUriResponse"]  as? [String: Any],
-        let uriReceived = response["Uri"] as? String else { return }
+        let uriReceived = response["Uri"] as? String else {
+            uri("")
```

Comment:
Is there a particular reason for returning an empty string vs return nil?

Reply:
Same as the comment above. it would require to switch to Optional to be able to return nil